src/modules/push-notifications/__mocks__/PushNotificationsManager.js

```diff
@@ -11,3 +11,7 @@ export const unsubscribeNews = async (prevCity: string, prevLanguage: string): P
 export const subscribeNews = async (newCity: string, newLanguage: string): Promise<void> => {
   console.debug(`Subscribed to ${newCity}-${newLanguage}-news topic!`)
 }
+
+export const checkPushNotificationPermission = async (): Promise<string> => {
```

Comment:
Will this stay like this? Or are there any plans to change this function?

Reply:
Some day in the future... or in a galaxy far far away... :D